spec/pluralrules.html

```diff
@@ -79,35 +75,42 @@ <h1>GetOperands ( _s_ )</h1>
              <td>Number of digits of [[Number]].</td>
```

Comment:
@sffc there are problems with the definitions of these fields:
* [[IntegerDigits]] is defined as "Number of digits of [[Number]]", but is populated with the _number value_ of the integer digits (e.g., it is equal to 8 for an input string like "8.540").
* [[FractionDigits]] and [[NumberOfFractionDigits]] have effectively the same definition, even though the former is populated with the number value of the fractional digits while the latter is populated with the _count_ of such digits (e.g., they are respectively equal to 540 and 3 for an input string like "8.540").
* [[FractionDigitsWithoutTrailing]] and [[NumberOfFractionDigitsWithoutTrailing]] also have effectively the same definition, even though the former is populated with the number value of the truncated fractional digits while the latter is populated with the _count_ of such digits (e.g., they are respectively equal to 54 and 2 for an input string like "8.540").

I'd like to fix this... do you know what the actual intent is?

Reply:
I think these are supposed to correspond to: https://unicode.org/reports/tr35/tr35-numbers.html#Operands

- i | integer digits of n.
- v | number of visible fraction digits in n, with trailing zeros.*
- w | number of visible fraction digits in n, without trailing zeros.*
- f | visible fraction digits in n, with trailing zeros.*
- t | visible fraction digits in n, without trailing zeros.*

Reply:
Thanks. I added references and updated the text to match (which is still just editorial work).